File: string_replace.py
# 文字列操作関数
def replace_strings(text, pattern, replacement):
    for _ in range(100000):
        text = text.replace(pattern, replacement)
    return text

# 大量の文字列を生成し操作
text = "A" * 1000000  # 100万文字の"A"
result = replace_strings(text, "A", "B")

File: string_replace.pl
#!/usr/bin/perl
use strict;
use warnings;

# 文字列操作関数
sub replace_strings {
    my ($text, $pattern, $replacement) = @_;
    $text =~ s/$pattern/$replacement/g for (1..100000);
    return $text;
}

# 大量の文字列を生成し操作
my $text = "A" x 1000000; # 100万文字の"A"
my $result = replace_strings($text, "A", "B");
